setup.py

```diff
@@ -22,7 +22,7 @@
         "Programming Language :: Python :: 3.7",
         "Framework :: Pytest",
     ],
-    description="Pytest plugin to update test results on a jUnit XML from previous run.",
+    description="Pytest plugin to update test results on a test report from previous run.",
```

Comment:
I think it is nice to specify the JUnit XML report, any reason you changed this?

Reply:
Nop. Better to be explict about it. Fixed.